src/sidepanel/components/case/PilotStepBodyDeclaration/index.tsx

```
@@ -0,0 +1,55 @@
+import { IPilotType } from "@/common/types/casePilot";
+import imgDeclaration from "@/resource/oss/assets/imgDeclaration.webp";
+
+interface PilotStepBodyDeclarationProps {
+  pilotInfo: IPilotType | null;
+}
+
+function PurePilotStepBodyDeclaration(props: PilotStepBodyDeclarationProps) {
+  const { pilotInfo } = props;
+
+  const handleBtnJumpClick = useCallback(async () => {
+    if (!!pilotInfo?.tabInfo?.id) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)